<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="QWidget" name="login_page">
       <widget class="QLabel" name="login_welcome_lbl">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#400080;&quot;&gt;请登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="login_loginBtn">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="login_signinBtn">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="login_buyer_rBtn">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>170</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>买家</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="login_seller_rBtn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>170</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>卖家</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="login_nameEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="login_pwdEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 密码：</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="myGoodsList_page"/>
      <widget class="QWidget" name="seller_mainPage">
       <widget class="QPushButton" name="goodsListBtn_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>查看商品列表</string>
        </property>
       </widget>
       <widget class="QPushButton" name="myGoodsListBtn">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>140</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>个人商品列表</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stockBtn">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>进货</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logoutBtn_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>240</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>注销</string>
        </property>
       </widget>
       <widget class="QPushButton" name="getInfoBtn_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>42</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>查看个人信息</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="charge_page">
       <widget class="QPushButton" name="charge_chargeBtn">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>120</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>充值</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="charge_numEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>121</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>充值金额</string>
        </property>
       </widget>
       <widget class="QPushButton" name="returnBtn_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="charge_pwdEdit">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>230</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>230</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>银行卡密码</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>银行卡</string>
        </property>
       </widget>
       <widget class="QComboBox" name="charge_comboBox">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>104</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="goodsList_page">
       <widget class="QListView" name="goodsList_list">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>40</y>
          <width>501</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="goodsList_searchEdit">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="goodsList_searchBtn">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
       <widget class="QPushButton" name="goodsList_addBtn">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>280</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>加入购物车</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="goodsList_amountEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>280</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="returnBtn_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>280</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 数目：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="goodsList_idEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>280</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>  商品id：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="goodsList_changeBtn">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>280</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>修改商品</string>
        </property>
       </widget>
       <zorder>label_5</zorder>
       <zorder>goodsList_list</zorder>
       <zorder>goodsList_searchEdit</zorder>
       <zorder>goodsList_searchBtn</zorder>
       <zorder>goodsList_addBtn</zorder>
       <zorder>goodsList_amountEdit</zorder>
       <zorder>returnBtn_2</zorder>
       <zorder>goodsList_idEdit</zorder>
       <zorder>label_11</zorder>
       <zorder>goodsList_changeBtn</zorder>
      </widget>
      <widget class="QWidget" name="info_page">
       <widget class="QTextBrowser" name="info_text">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>50</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="returnBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="stoke_page">
       <widget class="QPushButton" name="returnBtn_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="stoke_amountEdit">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>90</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="stoke_priceEdit">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>140</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="food_rBtn">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>185</y>
          <width>89</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>食品类</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="elet_rBtn">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>185</y>
          <width>89</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>电子产品类</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="living_rBtn">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>185</y>
          <width>89</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>生活用品类</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stoke_stokeBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>进货</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="stoke_nameEdit">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>40</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 商品名</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>商品数目</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>142</y>
          <width>54</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>商品单价</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="mDateEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>220</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="vDateEdit">
        <property name="geometry">
         <rect>
          <x>229</x>
          <y>220</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="disDateEdit">
        <property name="geometry">
         <rect>
          <x>389</x>
          <y>220</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>221</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>生产日期</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>163</x>
          <y>220</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 有效期</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>220</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 降价期</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="buyer_mainPage">
       <widget class="QPushButton" name="getInfoBtn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>查看个人信息</string>
        </property>
       </widget>
       <widget class="QPushButton" name="goodsListBtn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>查看商品列表</string>
        </property>
       </widget>
       <widget class="QPushButton" name="chargeBtn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>充值</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logoutBtn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>260</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>注销</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bindButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>210</y>
          <width>131</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>绑定银行卡</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cartBtn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>120</y>
          <width>131</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>查看购物车</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="cart_page">
       <widget class="QListView" name="cartList_list">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>461</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="returnBtn_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QPushButton" name="purchaseBtn">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>270</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>结算</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>  商品id：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="cartList_amountEdit">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="cartList_idEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>220</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 更新数目：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cartList_reAddBtn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>220</y>
          <width>61</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>更新</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="bind_page">
       <widget class="QLineEdit" name="bind_pwdEdit">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> 密码：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>80</y>
          <width>54</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>卡号：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bind_bindBtn">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>绑定</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="bind_numEdit">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
